core/src/main/java/org/geysermc/geyser/util/InventoryUtils.java

```diff
@@ -74,6 +74,10 @@ public class InventoryUtils {
      * each recipe needs a unique network ID (or else in .200 the client crashes).
      */
     public static int LAST_RECIPE_NET_ID;
+    /**
+     * Book cloning recipe ID; stored separately as its recipe works differently from others.
+     */
+    public static final int BOOK_CLONING_RECIPE_ID = 278;
```

Comment:
This can't be hardcoded, as it could differ between two sessions on standalone working with different recipes.